app/src/main/res/values/strings.xml

```diff
@@ -11,9 +11,12 @@
     <string name="login_progress_text">Loggin in...</string>
     <string name="login_incorrect_username">Username is incorrect</string>
     <string name="login_incorrect_password">Password is incorrect</string>
-
     <!--Strings related to create account-->
     <string name="login_invalid_email">Email or Username is invalid</string>
```

Comment:
Get rid of this string resource copied from the login feature and create resources related to Create Account's functionality.